pytest/tests/sanity/rosetta.py

```diff
@@ -315,47 +316,189 @@ def test(want_code, callback, *args, **kw) -> _Dict:
              block_id=block_1_id,
              trans_id=trans_0_id)
 
-    def test_delete_implicit_account(self) -> None:
+    def _get_account_balance(self,
+                             account: key.Key,
+                             require: bool = True) -> typing.Optional[int]:
```

Comment:
nit:

you can do at the top of the file:
from typing import Optional

Reply:
This is bigger discussion I suppose. Throughout all the code (in nearcore and nayduck) I’ve been using `typing.xxx` so this preserves that style. If we want to change this we should apply the style equally to all code in separate PR.